Tests/fixtures/smartphone.yml

```diff
@@ -2691,8 +2691,8 @@
     type: browser
     name: Aloha Browser
     version: "2.9.0.2"
-    engine: WebKit
-    engine_version: "537.36"
+    engine: Blink
+    engine_version: ""    
```

Comment:
end 4 spaces should be removed 'engine_version: ""
'

Reply:
Done